go.mod

```diff
@@ -11,27 +11,27 @@ replace github.com/spf13/cobra => github.com/spf13/cobra v1.4.0
 
 require (
 	github.com/ONSdigital/dp-api-clients-go v1.43.0 // indirect
-	github.com/ONSdigital/dp-healthcheck v1.5.0 // indirect
-	github.com/ONSdigital/dp-kafka/v2 v2.7.3
+	github.com/ONSdigital/dp-healthcheck v1.6.1 // indirect
+	github.com/ONSdigital/dp-kafka/v2 v2.8.0
 	github.com/ONSdigital/dp-net v1.5.0
 	github.com/ONSdigital/go-ns v0.0.0-20210916104633-ac1c1c52327e
```

Comment:
the indirect and actual seem to be mixed in ?

Reply:
This is how I picked it up from the main branch so I don't want to change it.